packages/button/src/ButtonBase.tsx

```diff
@@ -46,68 +49,65 @@ function getSizeCss(props: ButtonProps): string {
   }
 }
 
-const sizeCss = css<ButtonProps>`
-  ${getSizeCss}
-`
-
+/**
+ * Gets variant styles.
+ */
 function getVariantCss(props: ButtonProps): string {
+  /**
+   * {@link https://zeroheight.com/3ddd0f892/p/01a397-buttons/t/190120}
+   */
+  if (props.disabled) {
+    return `
+      background-color: ${grey20()};
+      color: ${grey40()};
+      &:focus {
+        outline: 0.2rem solid ${informative50()};
+        outline-offset: 0.2rem;
+      }
+      &:hover {
+        background-color: ${grey20()};
+        cursor: default;
+      }
+      &:active {
+        background-color: ${grey80()};
+      }
+    `
+  }
```

Comment:
@eeeeean I refactored disabled logic here since I saw it was redundant in primary/secondary

Reply:
Makes sense!